components/shared/embedder/resources.rs

```diff
@@ -55,16 +55,26 @@ pub fn sandbox_access_files_dirs() -> Vec<PathBuf> {
 }
 
 pub enum Resource {
+    /// A json file of [`Preferences`](servo_config::pref_util::Preferences) configuration.
+    /// It can be empty but lots of features will be disabled.
     Preferences,
     BluetoothBlocklist,
     DomainList,
+    /// A preload list of HTTP Strict Transport Security. It can be an empty list and
+    /// [`HstsList::default()`](net::hsts::HstsList) will be called.
```

Comment:
```suggestion
    /// A preloaded list of HTTP Strict Transport Security. It can be an empty list and
    /// [`HstsList::default()`](net::hsts::HstsList) will be called.
```